### PR TITLE
Do not autoformat on release

### PR DESCRIPTION
**Pull request by npepinpe**
## Description

Ran into this issue when releasing, where it will try to auto format during the release, resulting in changes preventing a release of taking place.

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to [backport](https://github.com/zeebe-io/zeebe/compare/stable/0.24...develop?expand=1&template=backport_template.md&title=[Backport%200.24]) the fix to the last two minor versions. You can trigger a backport by assigning labels (e.g. `backport stable/0.25`) to the PR, in case that fails you need to create backports manually.

Testing:
* [ ] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually
* [ ] The change has been verified by a QA run
* [ ] The impact of the changes is verified by a benchmark 

Documentation: 
* [ ] The documentation is updated (e.g. BPMN reference, configuration, examples, get-started guides, etc.)
* [ ] New content is added to the [release announcement](https://drive.google.com/drive/u/0/folders/1DTIeswnEEq-NggJ25rm2BsDjcCQpDape)

### CODE REVIEW

File: .ci/scripts/release/maven-release.sh [hunk 1]
```diff
@@ -11,4 +11,5 @@ mvn -s ${MAVEN_SETTINGS_XML} release:prepare release:perform -B \
     -DpushChanges=${PUSH_CHANGES} \
     -DremoteTagging=${PUSH_CHANGES} \
     -DlocalCheckout=${SKIP_DEPLOY} \
-    -Darguments='--settings=${MAVEN_SETTINGS_XML} -DskipTests=true -Dgpg.passphrase="${GPG_PASS}" -Dskip.central.release=${SKIP_DEPLOY} -Dskip.camunda.release=${SKIP_DEPLOY} -Dzbctl.force -Dzbctl.rootDir=${ZBCTL_ROOT_DIR}'
+    -P!autoFormat \
```
Comment (korthout) [inline .ci/scripts/release/maven-release.sh:14]: 💭 I'm not sure whether -P!autoFormat is necessary on the release:prepare and release:perform steps. I imagine that these do no run the entire execution lifecycle, but rather only those 2 steps. In any case, it doesn't do any harm either.

**Review by korthout — APPROVED** [review]
LGTM 🪴

**Comment by npepinpe** [comment]
bors merge

**Comment by npepinpe** [comment]
bors retry

**Pull request merged**